lib/chef-acceptance/chef_runner.rb

```diff
@@ -13,12 +13,14 @@ class ChefRunner
     attr_reader :test_suite
     attr_reader :recipe
     attr_reader :duration
+    attr_reader :timeout
 
-    def initialize(test_suite, recipe)
+    def initialize(test_suite, recipe, timeout)
       @test_suite = test_suite
       @acceptance_cookbook = test_suite.acceptance_cookbook
       @recipe = recipe
       @duration = 0
+      @timeout = timeout
```

Comment:
At this point I'd also be ok with an `CHEF_ACCEPT_TIMEOUT` env var.  So much code bloat for the timeout thing due to what I see as a little design conflict between application and chef_runner.